packages/rspack/src/builtin-plugin/lazy-compilation/backend.ts

```diff
@@ -82,16 +82,22 @@ const getBackend =
 		const listen =
 			typeof options.listen === "function"
 				? options.listen
-				: (server: Server) => {
-						let listen = options.listen;
-						if (typeof listen === "object" && !("port" in listen))
-							listen = { ...listen, port: undefined };
-						server.listen(listen);
-					};
+				: typeof options.server === "function"
+					? // if user offers custom server, no need to listen
```

Comment:
Can you update the documentation to tell this behavior?

Reply:
I think we are missing a lot of documentation on custom lazy compilation backends